### ISSUE TITLE
problem with cropping

### ISSUE THREAD

**Issue by stax76** [rigaya/NVEnc#10]
Hi, if I remember right then there was a cropping issue you said it's a SDK problem, maybe you can contact NVIDIA or abort with an error message.

https://github.com/stax76/staxrip/issues/204

**Comment by rigaya** [comment]
Thank you for reporting, regression should have been fixed with 3.13.

**Comment by stax76** [comment]
I verify that it's fixed, thanks!

**Comment by stax76** [comment]
It seems there is still something wrong, the crop values are doubled:

```
D:\Projekte\VS\VB\StaxRip\bin\Apps\NVEnc\NVEncC64.exe --cqp 20:23:25 --preset quality --crop 656,368,656,368 --output-res 1920x1080 -i "D:\Video\Samples\4K\The World in HDR.mkv" -o "D:\Video\Samples\4K\The World in HDR_temp\The World in HDR_new_out.h264"

NVEncC (x64) 3.17 (r566) by rigaya, Aug  1 2017 20:14:59 (VC 1900/Win/avx2)
OS Version     Windows 10 x64 (15063)
CPU            Intel Core i7-6700K @ 4.00GHz (4C/8T)
GPU            #0: GeForce GTX 1060 6GB (10 EU) @ 1771 MHz (384.94)
NVENC / CUDA   NVENC API 8.0, CUDA 9.0, schedule mode: sync
Input Buffers  CUDA, 36 frames
Input Info     avcuvid: VP9, 3840x2160, 60000/1001 fps
Vpp Filters    crop: 656,368,656,368/cspconv(nv12 -> yv12)
resize(spline36): 1216x688 -> 1920x1080
```

```
PS C:\Users\Frank> 3840 - 656 * 2
2528
PS C:\Users\Frank> 3840 - 656 * 4
1216
PS C:\Users\Frank>
```

**Comment by rigaya** [comment]
Thank you for reporting, will be fixed in next release.

**Comment by rigaya** [comment]
It has been fixed with NVEnc 3.20.

**Comment by stax76** [comment]
That's great, thanks.
